src/spec_procs.cpp

```diff
@@ -1153,6 +1196,10 @@ void train_attribute(struct char_data *ch, struct char_data *trainer, int ind, i
   // Apply the change.
   GET_REAL_ATT(ch, attr) += 1;
 
+  // Update channel points if in chargen
+  if (trainers[ind].is_newbie && IS_OTAKU(ch))
```

Comment:
You're probably going to have to change the logic so that you can't train/untrain at the newbie stat trainer if you have any channel skills set. Otherwise, someone could go spend all their points, then walk back to the trainer, UNTRAIN INT, TRAIN INT, and have max points again.